doc/whats_new.rst

```diff
@@ -2,6 +2,7 @@
 
 .. _whats_new:
 
+.. currentmodule:: mne_bids
```

Comment:
>  the linking of :func:foo also worked before

I put it here thinking it would fix some errors but it didn't, so indeed it turns out this was unnecessary, so I'll remove it.

Reply:
Never mind, won't remove it since this has been merged, but shouldn't be a problem to have it in there

Reply:
+1